packages/playwright-test/src/testInfo.ts

```diff
@@ -166,7 +164,7 @@ export class TestInfoImpl implements TestInfo {
     const description = modifierArgs[1];
     this.annotations.push({ type, description });
     if (type === 'slow') {
-      this.setTimeout(this.timeout * 3);
+      this._timeoutManager.setDefaultTimeout(this.timeout * 3);
```

Comment:
I'd expect that `test.slow()` inside fixture / hooks affect the context behavior similarly to `test.setTimeout`?